d2l-dom-focus.html

```diff
@@ -195,19 +195,49 @@
 
 		},
 
-		isFocusable: function(node, includeHidden) {
+		getPreviousFocusableAncestor: function(node, includeHidden, includeTabbablesOnly) {
+
+			if (!node) {
+				return null;
+			}
+
+			if (includeHidden === undefined) {
```

Comment:
Thoughts on initializing default value for `includeTabbablesOnly` as well?  Currently it would be `true` based on default value for `isFocusable`.